templates/content-no-footer-about-site-en.html

```diff
@@ -0,0 +1,18 @@
+---
+{
+	"title": "Content page - Hide 'About government' section and utility links from 'About this site',
```

Comment:
Missing a double quote at the end
```suggestion
	"title": "Content page - Hide 'About government' section and utility links from 'About this site'",
```